source/_integrations/watergate.markdown

```diff
@@ -0,0 +1,46 @@
+---
+title: Watergate
+description: Instructions on how to integrate Watergate within Home Assistant.
+ha_category:
+  - Sensor
+  - Valve
+  - Binary Sensor
+  - Number
+  - Water management
+ha_release: '2025.1'
+ha_iot_class: Local Push
+ha_config_flow: true
+ha_codeowners:
+  - '@adam-the-hero'
+ha_domain: watergate
+ha_platforms:
+  - valve
+---
+
+The **Watergate** {% term integration %} integrates your Watergate Devices (currently Sonic) to your Home Assistant.
+With this integration you are able to:
+
+- control your valve
+- configure auto shut off parameters
+- gather auto shut off parameters
+- check diagnostics of the device
```

Comment:
Please make sure the PR matches the current features implemented. As in, we should be able to cut a release at this moment and the documentation should be correct and not describing future features